jenkins-x-lint.yml

```diff
@@ -14,6 +14,15 @@ pipelineConfig:
                 value: http://jenkins-x-athens-proxy
               - name: PATH
                 value: "/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/workspace/go/bin"
+            options:
```

Comment:
@abayer, so you think the current issues with the `lint` checks drive from too few resources?

Do you think this error stems from there as well?

```
RUN ./hack/linter.sh
Installing GolangCI-Lint
golangci/golangci-lint info checking GitHub for tag 'v1.16.0'
golangci/golangci-lint info found version: 1.16.0 for v1.16.0/linux/amd64
golangci/golangci-lint info installed /workspace/go/bin/golangci-lint
level=info msg="[lintersdb] Active 6 linters: [deadcode errcheck gofmt gosec misspell unconvert]"
level=info msg="Memory: 1027 samples, avg is 68.9MB, max is 68.9MB"
level=info msg="Execution took 5m0.692742717s"
level=info msg="[loader] Go packages loading at mode load types and syntax took 4m59.772365495s"
level=error msg="Running error: context loading failed: no go files to analyze"
level=error msg="Deadline exceeded: try increase it by passing --deadline option"
```

I've seen it several times now.  For some reasons the linter does not seem to see the Go files!?

Reply:
Since it seems to be a timeout, given the 5m bit, it does feel correlated.